CMakeLists.txt

```
@@ -74,6 +74,7 @@ cmake_dependent_option(ENABLE_GLES "Enable OpenGL ES support" OFF "NOT ENABLE_EM
 option(ENABLE_BOOST_FILESYSTEM "Force the use of boost::filesystem, even if the compiler supports C++17." OFF)
 cmake_dependent_option(ENABLE_INSTALL "Enable installing projectM libraries and headers." OFF "NOT PROJECT_IS_TOP_LEVEL" ON)
 option(ENABLE_SYSTEM_GLM "Enable use of system-install GLM library" OFF)
+option(ENABLE_SYSTEM_PROJECTM_EVAL "Enable use of system-install projectM-eval library" OFF)
```

I'd actually prefer this if it's available. Linux distro s like Gentoo also ideally want this as separate package. So the build script should just look for it by default, giving the dev the opportunity to disable it and use the built-in version.

```suggestion
option(ENABLE_SYSTEM_PROJECTM_EVAL "Enable use of system-install projectM-eval library" ON)
```